site/cfg.php

```diff
@@ -1,11 +1,11 @@
 <?php
 require_once __DIR__ . '/vendor/autoload.php';
 
-if (is_dir("/home/gettauru")) {
```

Comment:
To make this work on both new & old, please change it into `if .. else if ... else` structure.

Reply:
fixed